src/sql/upgrade-76.sql

```diff
@@ -0,0 +1,12 @@
+-- We don't know if existing checkouts are deep clones.  This will
+-- force a new fetch (and most likely trigger a new build for deep
+-- clones, as the binary contents of '.git' are not deterministic).
+DELETE FROM CachedGitInputs;
```

Comment:
This freaked me out a bit but we already do this often: https://github.com/NixOS/hydra/blob/04fdb8e6e66d68d2625bdefa7f1c9d99e812c15b/src/lib/Hydra/Controller/Admin.pm#L51-L59